src/demo/demo.js

```diff
@@ -0,0 +1,103 @@
+console.log("demo.js")
+/*
+Importing index.js would work, but it would mean we would be bundling all that code into this demo bundle. Therefore we wouldn't be testing as
+close to production as with this approach. This approach will actually
+use the onfido bundle, the one that clients will use as well.
+ */
+const Onfido = window.Onfido
+
+const options = window.location
+                      .search.slice(1)
+                      .split('&')
+                      .reduce((/*Object*/ a, /*String*/ b) => {
```

Comment:
Why not `(acc, keyVal)` rather than `(a, b)`?

Reply:
code was there already like this, but it was actually me that brought this functionality many moons ago. I think it's because it's copy pasta.